config/express.js

```diff
@@ -3,7 +3,7 @@ var passport = require('passport'),
 
 var verifyHandler = function (adminToken, token, tokenSecret, profile, done) {
   process.nextTick(function() {
-    User.findOne({name: profile.name}, function(err, user) {
+    User.findOne({id: name}, function(err, user) {
```

Comment:
ID is the primary key in mongo, that's how it works. So yes, it only works here because this is a query, but that's the point. We should always use the primary key when we are using the primary key.

But yes, I meant to do profile.name, that was a mistake.